### ISSUE TITLE
Log created in the working dir although -config parameter was specified

### ISSUE THREAD

**Issue by mauromol** [matteobaccan/html2pop3#49]
I downloaded html2pop3 3.0.4 and successfully configured it and run in Linux Mint 19.3 XFCE.
I then added it to the commands to launch on user login. In this context, the working dir seems to be set to the user home directory. Then, looking at the source, I saw the -config parameter, so I passed the command line parameter `-config .config/html2pop3/config.cfg`.
First of all: I found no documentation about this useful parameter.
Second: I see I must specify the path to the config file, not just to the owning directory: it would be more useful to specify the directory containing configuration files, since I put there other files as well (like nntp.cfg, rss.cfg, etc.). I tried to specify `-config .config/html2pop3`, but it doesn't work as expected.
Third: now if I right click on html2pop3 tray icon and select to view the log, it says:

```
Log file: /home/mauro/.config/html2pop3/html2pop3.log
Configuration file: /home/mauro/.config/html2pop3/config.cfg
```

However, the log file is still created as /home/mauro/html2pop3.log and not in the mentioned position.

**Comment by matteobaccan** [comment]
First of all: sorry for the late reply

I'm working to improve the code of HTML2POP3: I'have migrated from the old SourceForge and some parties of HTML2POP3 are not converter: for example documentation. I'm working on it.

You are right: -config is undocumented at the moment and is the absolute path of `config.cfg`

I can add a `-configdir` that may be more useful in this situation, but my idea is to put all the configuration file path inside `config.cfg`. At the moment the situation is not clear and for this reason, the parameter is undocumented.

The third is a bug, I'll fix this soon and improve `config.cfg`

I'll update soon the code in `master ` branch

**Comment by matteobaccan** [comment]
Hi

I have fixed all in 3.0.6 release

-config specified the config.cfg with an absolute path. This path is also used to getting all other .cfg. Now all the paths of config files are printed inside the log
- the GUI now shows the correct path of html2pop3.log
All fixes are in master and will be available in 3.0.6 release

all the best
mate baccan
